examples/problem_decomposition/mifno.ipynb

```diff
@@ -28,11 +28,13 @@
     "    !pip install pyscf\n",
     "    !pip install git+https://github.com/goodchemistryco/Tangelo.git@develop --quiet\n",
     "\n",
-    "# Download the data folder at https://github.com/goodchemistryco/Tangelo/branches/develop/tangelo/problem_decomposition/tests/incremental/data\n",
+    "# Download the data folder at https://github.com/goodchemistryco/Tangelo/branches/develop/tangelo/problem_decomposition/tests/incremental/data/BeH2_CCPVDZ_MIFNO_HBCI\n",
```

Comment:
Tests are currently failing, your svn install command did not go through. I'm not sure we need svn btw. Have you checked if any other notebook is grabbing data from the repo and if yes how are they doing it?

Whatever works tbh, in the end.

Reply:
This [PR](https://github.com/goodchemistryco/Tangelo/pull/351) was supposed to be merged before this one.